event_mail/tests/test_event_mail.py

```diff
@@ -86,7 +87,7 @@ def test_event_template_no_config(self):
         event = self.env["event.event"].create(vals)
         self.assertEqual(
             len(event.event_mail_ids),
-            0,
+            3,
             "Event Mail: mails scheduler no created for this event",
```

Comment:
But then, was the test wrong before? :thinking:

Reply:
Here the default value of the `event_mail_ids` field in events is being testing when no template is defined in the res.config.setting. In v13, the event was created with an empty `event_mail_ids`, but in v15, it takes what the `_default_event_mail_type_ids` method of `event.type` returns  https://github.com/OCA/OCB/blob/15.0/addons/event/models/event_event.py#L27 and creates 3 records for `event_mail_ids`.